app/models/concerns/push_updates_for_message.rb

```diff
@@ -42,7 +42,7 @@ def update_client_store
 
   # All reviewers/supervisors/order_fulfillers
   def notify_deletion_to_subscribers
-    if object_class == "Order"
+    if ["Order", "Package"].include?(object_class)
       channels = [Channel::ORDER_FULFILMENT_CHANNEL]
     else # Offer/Item
       channels = [Channel::REVIEWER_CHANNEL, Channel::SUPERVISOR_CHANNEL]
```

Comment:
Can use Inventory channel for Packages now (stock app)

```suggestion
    case object_class
    when "Order"
      channels = [Channel::ORDER_FULFILMENT_CHANNEL]
    when "Package"
      channels = [Channel::INVENTORY_CHANNEL]
    else # Offer / Item
      channels = [Channel::REVIEWER_CHANNEL, Channel::SUPERVISOR_CHANNEL]
```

Stock app listens on both Inventory Channel and OrderFulfilment channel so this change shouldn't make any actual difference but it's good to have it using the right channel for future work.